uiowa/uiowa
# Grad College Thesis Defense Views

Resolves https://github.com/uiowa/uiowa/issues/2836
Resolves https://github.com/uiowa/uiowa/issues/2837

## To Test

Create thesis defense content (past dates, future dates) (https://grad.local.drupal.uiowa.edu/node/add/thesis_defense) and view the following pages: 

https://grad.local.drupal.uiowa.edu/academics/thesis-and-dissertation/list (should only show upcoming)
https://grad.local.drupal.uiowa.edu/academics/thesis-and-dissertation/calendar (items should render on calendar)
